spec/helpers/post_location_helper_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe PostLocationHelper, :type => :helper do
+  describe "#link_to_post" do
+    it "returns gamedev url" do
+      post = FactoryGirl.create(:external_post)
+      expect(helper.link_to_post(post)).to  match("http://gamedev.stackexchange.com/")
+    end
+
+    it "returns posts url" do
+      post = FactoryGirl.create(:internal_post)
+      expect(helper.link_to_post(post)).to  match("/posts/")
+    end
+  end
+
+  describe "#show_tags" do
+    it "return questions tags" do
+      post = FactoryGirl.create(:external_post)
+      binding.pry
+      expect(helper.show_tags(post)).to  match("question")
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.